web/src/main/webapp/xsl/conversion/import/EsriGeosticker-to-ISO19139.xsl

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - 
+
+Transformation to produce ISO19139 documents from input documents that
+combine "extra" elements (Esri-specific or from ISO19115 DTD) with an included
+MD_Metadata element.
+
+The current version ignores all elements except for MD_Metadata, which is copied to the output verbatim.
+
+author Heikki Doeleman
+- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
+<xsl:stylesheet version="2.0"
+xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+xmlns:xs="http://www.w3.org/2001/XMLSchema"
+xmlns:fn="http://www.w3.org/2005/xpath-functions"
+xmlns:gmd="http://www.isotc211.org/2005/gmd">
+    <xsl:output method="xml" version="1.0" encoding="UTF-8" indent="yes"/>
+
+  <xsl:strip-space elements="*" />
+
+  <xsl:param name="publishLocalResources">False</xsl:param>
+
+  <xsl:variable name="smallcase" select="'abcdefghijklmnopqrstuvwxyz'" />
+  <xsl:variable name="uppercase" select="'ABCDEFGHIJKLMNOPQRSTUVWXYZ'" />
+
+  <!-- the input document is supposed to have either <metadata> or <Metadata> as root element -->
+  <xsl:template match="/metadata|/Metadata">
+    <xsl:apply-templates/>
+  </xsl:template>
+
+  <!-- the wrapped <MD_Metadata> is copied in its entirety to the output -->
+  <!--<xsl:template match="gmd:MD_Metadata|MD_Metadata">
+    <xsl:copy-of select="."/>
+  </xsl:template>-->
+
+  <xsl:template match="gmd:MD_Metadata|MD_Metadata">
+    <xsl:copy>
+      <xsl:apply-templates mode="iso" select="@*"/>
+      <xsl:apply-templates mode="iso" select="gmd:fileIdentifier|fileIdentifier"/>
```

Comment:
Can't be simplified with the following?

`<xsl:apply-templates mode="iso" select="*"/>`

Or `gmd:MD_Metadata` can contain non-iso elements?

Reply:
the original author, Heikki, is not available anymore, not sure why he introduced it, afaik md_metadata would not contain any other elements to be stripped

Reply:
There is room for improvements (eg. use lower-case fn, remove ref to localhost/127.0.0.1, probably publishLocalResources can't be used in the context of GN import, remove useless comment, formatting) but if it can help GeoSticker user (I'm not sure what it is:)) to load their records, probably we can add it as it is ? @josegar74 what do you think ?